tests/functional/testUtapiClient.js

```diff
@@ -49,6 +106,53 @@ describe('Counters', () => {
                 next),
             next => utapiClient.pushMetricCreateBucket(reqUid, testBucket,
                 next),
-        ], () => _assertCounters(testBucket, done));
+        ], () => _assertCounters(testBucket, {
+            storageUtilized: 0,
+            numberOfObjects: 0,
+        }, done));
     });
+
+    it('should update counters to correct values < 0: putObject', done =>
```

Comment:
Since we are technically not updating counters but actually recording the counter for a timestamp, the test description should reflect that.